lib/WebService/Stripe.pm

```diff
@@ -173,6 +173,12 @@ method cancel_transfer(Str $id, :$headers) {
     return $self->post("/v1/transfers/$id/cancel", undef, headers => $headers);
 }
 
+method create_reversal($xfer_id, HashRef $data = {}, HashRef $opts = {}) {
```

Comment:
Something in LWP::UserAgent-land normalizes headers to lower-cased / underscore-separated, so "Stripe-Account", "stripe_AcCoUnT", and "stripe_account" are all equivalent.